### PR TITLE
refactor: commands response

### PR DESCRIPTION
**Pull request by anhnh12**
**What does this PR do?**
-   [x] Update commands response: https://www.notion.so/console-labs/Check-UI-Command-Response-8b18ac41cd3a4b199f964ee3c1089108

### CODE REVIEW

File: src/commands/config/log/info.ts [hunk 1]
```diff
@@ -1,45 +1,56 @@
...
+      author: ["Log channel", getEmojiURL(tripodEmojis.POD)],
+      description: `No logging channel configured for this guild.\nSet one with \`${PREFIX}log set <channel>.\``,
+    })
+    return { messageOptions: { embeds: [embed] } }
+  }
+
+  const embed = composeEmbedMessage(msg, {
+    author: ["Log channel", getEmojiURL(tripodEmojis.POD)],
+    description: `All activities will be monitored in <#${guildCfg.log_channel}>. To change this channel, run \`$log set #<channel>\`.`,
+    // description: `<#${guild.log_channel}> is currently monitored.\nTo change the monitored channel, run \`${PREFIX}log set <channel>.\``,
```
Comment (tuanddd) [inline src/commands/config/log/info.ts:42]:
```suggestion
```

File: src/commands/config/log/info.ts [hunk 2]
```diff
@@ -1,45 +1,56 @@
...
+    return {
+      messageOptions: {
+        embeds: [
+          getErrorEmbed({
+            msg,
+            description: "This command must be run in a Guild",
+          }),
+        ],
+      },
+    }
```
Comment (tuanddd) [inline src/commands/config/log/info.ts:25]: throw GuildIdNotFound

File: src/commands/config/log/info.ts [hunk 3]
```diff
@@ -1,45 +1,56 @@
...
-      return { messageOptions: { embeds: [embed] } }
-    }
-
-    const embed = composeEmbedMessage(msg, {
-      author: [msg.guild.name, msg.guild.iconURL() ?? ""],
-      description: `<#${guild.log_channel}> is currently monitored.\nTo change the monitored channel, run \`${PREFIX}log set <channel>.\``,
-    })
-    return { messageOptions: { embeds: [embed] } }
-  },
+  run: (msg) => runLogInfo({ msg, guild: msg.guild ?? undefined }),
```
Comment (tuanddd) [inline src/commands/config/log/info.ts:53]:
```suggestion
  run: (msg) => runLogInfo({ msg, guild: msg.guildId }),
```

File: src/commands/config/log/info.ts [hunk 4]
```diff
@@ -1,45 +1,56 @@
...
+import { getEmojiURL, tripodEmojis } from "utils/common"
 import { PREFIX } from "utils/constants"
 import { composeEmbedMessage, getErrorEmbed } from "utils/discordEmbed"
 
+export async function runLogInfo({
+  msg,
+  guild,
+}: {
+  msg?: Message
+  guild?: Guild
```
Comment (tuanddd) [inline src/commands/config/log/info.ts:13]:
```suggestion
  guildId?: number | null
```

File: src/commands/config/log/info.ts [hunk 5]
```diff
@@ -1,45 +1,56 @@
...
+}) {
+  if (!guild?.id) {
```
Comment (tuanddd) [inline src/commands/config/log/info.ts:15]:
```suggestion
  if (!guildId) {
```

File: src/commands/config/log_slash/info.ts [hunk 6]
```diff
@@ -1,44 +1,11 @@
...
-    return { messageOptions: { embeds: [embed] } }
-  }
-
-  const embed = composeEmbedMessage(null, {
-    author: [interaction.guild.name, interaction.guild.iconURL() ?? ""],
-    description: `Current monitoring channel is <#${guild.log_channel}>.\nYou can update using \`${SLASH_PREFIX}log set <channel>.\``,
-    originalMsgAuthor: interaction.user,
-  })
-  return { messageOptions: { embeds: [embed] } }
+  return await runLogInfo({ guild: interaction.guild ?? undefined })
```
Comment (tuanddd) [inline src/commands/config/log_slash/info.ts:10]:
```suggestion
  return await runLogInfo({ guild: interaction.guildId })
```

File: src/commands/defi/deposit.ts [hunk 7]
```diff
@@ -38,7 +43,8 @@ async function deposit(msg: Message) {
       messageOptions: {
         embeds: [
           composeEmbedMessage(msg, {
-            description: `:information_source: Info\n<@${msg.author.id}>, your deposit address has been sent to you via a DM`,
+            author: ["Deposit tokens", getEmojiURL(emojis.WALLET)],
+            description: `<@${msg.author.id}>, your deposit address has been sent to you. Check your DM!`,
```
Comment (tuanddd) [inline src/commands/defi/deposit.ts:47]:
```suggestion
            description: `${msg.author}, your deposit address has been sent to you. Check your DM!`,
```

File: src/commands/defi/withdraw.ts [hunk 8]
```diff
@@ -99,7 +97,8 @@ const command: Command = {
       msg.reply({
         embeds: [
           composeEmbedMessage(msg, {
-            description: `:information_source: Info\n<@${msg.author.id}>, a withdrawal message has been sent to you via a DM`,
+            author: ["Withdraw tokens", getEmojiURL(emojis.WALLET)],
+            description: `<@${msg.author.id}>, a withdrawal message has been sent to you. Check your DM!`,
```
Comment (tuanddd) [inline src/commands/defi/withdraw.ts:101]:
```suggestion
            description: `${msg.author.id}, a withdrawal message has been sent to you. Check your DM!`,
```

File: src/errors/CommandError.ts [hunk 9]
```diff
@@ -16,7 +16,7 @@ export class CommandError extends BotBaseError {
     description?: string
   }) {
     super(message)
-    this.name = "Command error"
+    this.name = "Internal error"
```
Comment (tuanddd) [inline src/errors/CommandError.ts:19]:
```suggestion
    // TODO: rename CommandError class to InternalError class
    this.name = "Internal error"
```

File: src/utils/wrapError.ts [hunk 10]
```diff
@@ -59,6 +59,7 @@ export async function wrapError(
         error.handle?.()
         ChannelLogger.alert(message, error).catch(catchAll)
       } else if (message.isCommand()) {
+        console.log({ error })
```
Comment (tuanddd) [inline src/utils/wrapError.ts:62]:
```suggestion
```

**Review by tuanddd — CHANGES_REQUESTED** [review]

**Review by trkhoi — APPROVED** [review]

**Review by tuanddd — APPROVED** [review]

**Pull request merged**